public/JavaScript/gallery.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let selectedAnimalType = null
+
+  // Function to hide all gallery items
+  const hideAllAnimals = () => {
```

Comment:
This function can receive an argument instead of searching form it inside the function. (Dependency injection)

This way the function doesn't require anything else, can be easily tested etc